app/objects/c_operation.py

```diff
@@ -232,6 +237,9 @@ def report(self, output=False, redacted=False):
                 await asyncio.sleep(10)
                 await self._update_operation(services)
                 await self._run_phases(services, planner)
+                if len(self.adversary_queue) > 0:
```

Comment:
this is a good thought. but there will be a few issues on the backend, mainly around the fact that this new adversary will only roll through the phases 1 time vs the operation periodically running the phases in case there is new information. instead of this, i would have the "update_operation" merge the new adversary into the existing adversary, that way all phases get picked up automatically.

Reply:
i think your best bet is to split this into 2 PRs:
1) make the hard-coded "adversary_id" property on c_operation a list instead of a string. deal with any of the fallout.
2) add your UI component to append to this list of adversary_ids during a running operation.